### PR TITLE
Alberto/atqr 128 create notifications templatedatatypes

### PR DESCRIPTION
**Pull request by albnunes**
this draft is how I implemented the template message. I would like you to point out to me the errors and bad practices in this code

### CODE REVIEW

**Comment by yfernandes** [comment]
Please be more specific when you ask for help on a draft.

File: libs/atqr/domain/src/lib/EmailService/templates/templateMessage.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { ChallengeAndInviteeEnvelope, ChallengeEnvelope } from '../dtos';
+import { ITemplateMessage } from './interfaces/templateMessage.interface';
+
+export enum NotificationChannel {
+  Email = 'Email',
+  SMS = 'SMS',
+  Push = 'Push',
+}
+// create an enum for each type of content
+export enum ChallengeEnum {
```
Comment (albnunes) [inline libs/atqr/domain/src/lib/EmailService/templates/templateMessage.ts]: I don't know if this implementation is the right one or the most efficient one to relate the emails to the content they should contain. @yfernandes

**Comment by albnunes** [comment]
only the last modification counts, the last ones were implementing completely wrong. @yfernandes

**Review by yfernandes — APPROVED** [review]
All basic, patterns and guidelines checks are passing. Everything looks correct and it's running.

**Pull request merged**